app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -227,6 +239,31 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
     fullNameTextView.text = resources.getString(R.string.patientsummary_name, name, genderLetter, age)
   }
 
+  private fun displayBpPassport(bpPassport: Optional<BusinessId>, isPhoneNumberVisible: Boolean) {
+    bpPassportTextView.visibility = if (bpPassport.isNotEmpty()) View.VISIBLE else View.GONE
+
+    bpPassportTextView.text = when {
+      bpPassport.isNotEmpty() -> {
+        val bullet = if (isPhoneNumberVisible) Unicode.bullet else ""
+        val identifier = bpPassport.toNullable()?.identifier!!
+        val textSpan = TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body2Left_White72)
```

Comment:
We do not need to explicitly set this text appearance here since we have already set it in the layout.